gallery/src/GalleryNav/GalleryNav.tsx

```diff
@@ -21,43 +21,77 @@ import {
   complement,
   values
 } from 'ramda';
-import { NxTreeView, NxTreeViewChild, NxFilterInput, NxTreeViewProps } from '@sonatype/react-shared-components';
+import {
+  NxCollapsibleItems,
+  NxFilterInput,
+  NxCollapsibleItemsProps,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
 
-import pageConfig from './pageConfig';
+import pageConfig from '../pageConfig';
 import { useLocation } from 'react-router';
 import { NavLink } from 'react-router-dom';
-import { PageMapping, PageConfig } from './pageConfigTypes';
-import { markByFilter, matchesFilter } from './filterUtil';
-
-const renderLinks = (filter?: string) => pipe(
-    keys,
-    map((pageName: string) =>
-      <NxTreeViewChild key={pageName}>
-        <NavLink to={`/pages/${pageName}`} activeClassName="selected">
-          {markByFilter(filter, pageName)}
-        </NavLink>
-      </NxTreeViewChild>
-    )
-);
+import { PageMapping, PageConfig, PageType, PageContentDescription } from '../pageConfigTypes';
+import { markByFilter, matchesFilter } from '../filterUtil';
+
+import { faFile, faCode, faRulerCombined } from '@fortawesome/free-solid-svg-icons';
+import { faReact, faSass, faJs } from '@fortawesome/free-brands-svg-icons';
+
+import './GalleryNav.scss';
+
+const PAGE_TYPE_TO_ICON_MAP: { [page in PageType]: JSX.Element } = {
+  'documentation': <NxFontAwesomeIcon icon={faFile} fixedWidth />,
+  'react': <NxFontAwesomeIcon icon={faReact} fixedWidth />,
+  'html': <NxFontAwesomeIcon icon={faCode} fixedWidth />,
+  'layout': <NxFontAwesomeIcon icon={faRulerCombined} fixedWidth />,
+  'sass': <NxFontAwesomeIcon icon={faSass} fixedWidth />,
+  'js': <NxFontAwesomeIcon icon={faJs} fixedWidth />
```

Comment:
The only differences between any of these is the `icon` prop value.  It'd be cleaner if the map just stored those values and the `NxFontAwesomeIcon` construction was down in the rest of the JSX

Reply:
> The only differences between any of these is the `icon` prop value. It'd be cleaner if the map just stored those values and the `NxFontAwesomeIcon` construction was down in the rest of the JSX

There were thoughts of potentially adding custom icons. I cleaned it up the construction here: 9c6c8c669e68b04d9b971e45aa6b13538d285c44